modules/install-central-services-helm-chart.adoc

```diff
@@ -18,3 +18,8 @@ $ helm install -n stackrox --create-namespace \
   -f <path_to_values_public.yaml> -f <path_to_values_private.yaml> <1>
 ----
 <1> Use the `-f` option to specify the paths for your YAML configuration files.
+
+[NOTE]
+====
+Optional: If using declarative configuration, add `-f <path_to_declarative-config-values.yaml` to this command to mount the declarative configurations file in Central.
```

Comment:
There is a Central system component that has a custom resource for it, so sometimes the doc is talking about the system component/object and sometimes it is talking about the custom resource. I think normally in the docs, "Central" isn't put in backticks, but I did for this section because it's referring to the custom resource. Not sure if there's another approach that would be less confusing.

Reply:
I don't think we should change this to custom resource, as this documentation is about the Helm installation.

Within Helm's context, the custom resource doesn't exist, so it'd be more confusing to me to mention it here.